lib/config/wrapper.go

```diff
@@ -105,6 +107,27 @@ type wrapper struct {
 	requiresRestart uint32 // an atomic bool
 }
 
+func (w *wrapper) StunServers() []string {
+	var addresses []string
+	for _, addr := range w.cfg.Options.StunServers {
+		switch addr {
+		case "default":
+			addresses = append(addresses, DefaultStunServers...)
+		default:
+			addresses = append(addresses, addr)
+		}
+	}
+
+	addresses = util.UniqueStrings(addresses)
+
+	// Shuffle
+	rand.Shuffle(len(addresses), func(i, j int) {
```

Comment:
This will be deterministic, mostly, as we don't seed the default random generator afaik. Maybe create a new `Rand` based on our `secureSource` and shuffle on that. Overkill, but at least [guaranteed to be random](https://xkcd.com/221/).